.github/workflows/flows.yml

```diff
@@ -0,0 +1,58 @@
+# Run Playwright user flow integration tests
+name: Flows
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  # push:
+  #   branches:
+  #     - main
+  pull_request:
+    branches:
+      - main
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
I'm fairly new to GitHub Actions, so forgive me if this is wrong, but does this property need a value after it?

Reply:
Nope, there's an optional inputs property though; we don't need it now. Can't manually dispatch until this file is in `main` though.

https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
YAML is even weirder than I thought! Works for me 👍🏻